src/app/features/course/course.module.ts

```diff
@@ -0,0 +1,15 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CourseComponent } from './course.component';
+
+
+
+@NgModule({
+  declarations: [
+    CourseComponent
+  ],
+  imports: [
+    CommonModule
```

Comment:
![image](https://user-images.githubusercontent.com/33813087/151662119-bb4a64da-d038-439e-a262-f233a4b98319.png)

Reply:
it is not resolved as I see